src/include/network/lan/address.rb

```diff
@@ -1430,9 +1424,7 @@ def AddressDialog
       end
 
       # proceed with WLAN settings if appropriate, #42420
-      if ret == :next && LanItems.type == "wlan" && LanItems.alias == ""
```

Comment:
So, aliases are not important anymore, could it arise in some problem?

Reply:
```LanItems::aliases``` is still in game. ```LanItems::alias``` is not. Looks confusing? That happens when someone re-implements a new feature and do not clean the code properly.